### PR TITLE
UI: Beautify add, edit and show

### PR DESCRIPTION
**Pull request by NightRaven49**
Addresses #559.

### CODE REVIEW

**Comment by tjtanjin** [comment]
Note: need to remove the information of those data (from textarea) that has now been moved into the tags.

**Comment by NightRaven49** [comment]
hmm, in that case i can see the point of beautifying for add and edit, but it would make show a redundant command imo, since the result would be indistinguishable from run/send on first inspection.

**Comment by tjtanjin** [comment]
Try stashing your changes and running `add`, `edit` and `show`. They reveal more details than the `send` and `run` commands such as `header`, `data` and `tag`. `show` is suppose to show every single detail of the endpoint whereas `send` and `run` only shows the endpoint and response fields.

File: src/main/java/seedu/us/among/ui/ResultDisplay.java [hunk 1]
```diff
@@ -144,10 +144,65 @@ public String getColorCode(String statusCode) {
...
+                result = result + feedback.substring(headerStart, meta);
+            }
+        }
+        if (tagStart > 0) {
+            result = result + "Tags:\n" + dividers + feedback.substring(tagStart + 6, meta);
+        }
+        result = result + "Last Saved Response:\n" + dividers + feedback.substring(response + 15);
+        return result;
+    }
+
```
Comment (ong6) [inline src/main/java/seedu/us/among/ui/ResultDisplay.java:205]: this beatify implementation does not hold if we change the toString output of feedback, endpoint or anything related to these comments. Hence I will have to reject this implementation of beautify.

**Review by ong6 — COMMENTED** [review]
Not sure if this implementation even stable

**Comment by NightRaven49** [comment]
Resolved with #572.

**Pull request closed without merging**